train.py

```diff
@@ -968,7 +968,7 @@ def restore_parts(path, model):
         else:
             log_event_path = "log/run-test" + str(datetime.now()).replace(" ", "_")
     print("Log event path: {}".format(log_event_path))
-    writer = SummaryWriter(log_dir=log_event_path)
+    writer = SummaryWriter(logdir=log_event_path)
```

Comment:
This will be incompatible to the older version of tensorboardX. How about `SummaryWriter(log_event_path)`?

Reply:
good call, that works for me